### PR TITLE
Fix typo in comment

### PR DESCRIPTION
**Pull request by chestack**
**What this PR does / why we need it**:
fix typo

**Release note**:

```release-note
None

### CODE REVIEW

**Comment by FengyunPan** [comment]
/lgtm
Welcome!

File: pkg/kubelet/kuberuntime/kuberuntime_manager.go [hunk 1]
```diff
@@ -614,7 +614,7 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, _ v1.PodStatus, podStat
 	}
 
 	// Keep terminated init containers fairly aggressively controlled
-	// This is an optmization because container removals are typically handled
+	// This is an optimization because container removals are typically handled
```
Comment (nikhita) [inline pkg/kubelet/kuberuntime/kuberuntime_manager.go]: This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-8da51ab012ecbdf2b013ecbcf1c37033.

Closing.

/close

**Pull request closed without merging**